js/homeview.js

```diff
@@ -732,7 +732,7 @@ enyo.kind({
 
 	rendered: function() {
 		this.inherited(arguments);
-		this.localize();
+		this.localize;
```

Comment:
Very strange fix: removing parenthesis is equivalent to don't call the function at all.